### PR TITLE
Billing Documentation

### PR DESCRIPTION
**Pull request by quannhoang**
1. Add instructions to add, configure and deploy billing infrastructures
2. Add role/access/permission needed for each cloud
3. Add billing diagrams
4. Change structure of Infrastructure part

### CODE REVIEW

**Comment by giraffesyo** [comment]
Exhaustive list of all broken links found:

- On source page path = /deploying-pw-infrastructure/base-infrastructure/creating-base-infra:
   -> linking to /managing-organizations/deploying-base-infrastructure/adding-cloud-accounts
   -> linking to /managing-organizations/deploying-base-infrastructure/configuring-base-infra

File: docs/managing-organizations/deploying-pw-infrastructure/billing-infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# About Billing Infrastructure
+
+CSPs provide resource usage and cost reports to customers at set intervals. This data is normally referred to as true cost data. Each CSP's cost export feature ultilizes unique cloud infrastructure and has a unique data format. To enable CSP billing data export, users have to follow the CSP's guide to create sufficient infrastructures and services.
```
Comment (giraffesyo) [inline docs/managing-organizations/deploying-pw-infrastructure/billing-infrastructure/README.md:3]: By who?

File: docs/managing-organizations/deploying-pw-infrastructure/billing-infrastructure/creating-billing-infra.md [hunk 2]
```diff
@@ -35,7 +31,7 @@ Click **+ Create Infrastructure**.
 ![create_billing_infra_3_mar23.png](/screenshots/create_billing_infra_3_mar23.png)
 <br />
 
-Next, you'll configure the infrastructure. This process looks slightly different for each cloud service provider; please see [**Configuring a Billing Infrastructure**](/managing-organizations/deploying-pw-infrastructure/billing-infrastructure/configuring-billing-infra) for more information .
+Next, you'll configure the infrastructure. This process looks slightly different for each cloud service provider; please see [**Configuring Billing Infrastructure**](https://docs.parallel.works//managing-organizations/deploying-pw-infrastructure/billing-infrastructure/configuring-billing-infra) for more information .
```
Comment (giraffesyo) [inline docs/managing-organizations/deploying-pw-infrastructure/billing-infrastructure/creating-billing-infra.md]: we should use relative links for anything that is in the docs

File: docs/managing-organizations/deploying-pw-infrastructure/billing-infrastructure/README.md (hunk 1 above)
Comment (quannhoang) [reply docs/managing-organizations/deploying-pw-infrastructure/billing-infrastructure/README.md]: by us, ill update to include PW.

**Review by giraffesyo — APPROVED** [review]

**Pull request merged**